apps/src/p5lab/AnimationPicker/AnimationPickerBody.jsx

```diff
@@ -101,19 +156,9 @@ class AnimationPickerBody extends React.Component {
   }
 
   render() {
```

Comment:
NIT: Since this is accessing the state and props variables more than once, consider updating it to use this pattern:
const {spritelab} = this.props;
const {categoryQuery, searchQuery, results} = this.state;